english/kill-rats/kill-rats.md

```diff
@@ -0,0 +1 @@
+You start to smash the rats with your feets and to punch the ones who jump to you, but no matter how bravely you fight they start to eat your body until your end...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.